MetaMask/metamask-extension
# fix: use svg for blockaid what's new instead of PNG

## **Description**
Replace blockaid what's new PNG image with SVG

## **Manual testing steps**

1. Delete MetaMask
2. Start MetaMask with `yarn start:flask`
3. Install MetaMask
5. You should see blockaid what's new icon

## **Screenshots/Recordings**
<img width="515" alt="Screenshot 2023-09-29 at 14 17 36" src="https://github.com/MetaMask/metamask-extension/assets/44811/cbe8e34d-ba13-449e-98e3-8d392e7d91cc">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
